chia/data_layer/s3_plugin_config.yml

```
@@ -5,12 +5,13 @@ instance-1:
     secret_access_key: "xxx"
     region: "xxx"
 
-  store_ids: ["xxx"]
-
-  urls: [ "xxx"]
-
-  buckets:
-    chia-datalayer-test-bucket-2: ["xxx"]
+  store_ids:
```

At this point, maybe just stores?

```suggestion
  stores:
```